packages/charts/react-charting/src/components/GroupedVerticalBarChart/GroupedVerticalBarChart.base.tsx

```diff
@@ -67,10 +75,10 @@ export interface IGroupedVerticalBarChartState extends IBasestate {
   calloutLegend: string;
 }
 
-export class GroupedVerticalBarChartBase extends React.Component<
-  IGroupedVerticalBarChartProps,
-  IGroupedVerticalBarChartState
-> {
+export class GroupedVerticalBarChartBase
+  extends React.Component<IGroupedVerticalBarChartProps, IGroupedVerticalBarChartState>
+  implements IChart
```

Comment:
>IChart [](http://example.com/codeflow?start=13&length=6)

can we make Chart a concrete class and give it the resposibility to hold the component ref. So that each chart do not need to do it.

Reply:
Yes, I think we can do that, but I will need to give it a try.

Reply:
We will explore this as follow up. We might have to use mixin to combine classes.